Issue with Babel 6 upgrade

Forgot to try https://github.com/sindresorhus/ava/commit/9e6d9724d52582714804bf7c8ec71bcd7c4af2ed on an actual project and there's an issue.

It seems Babel 6 tries to read the plugins from CWD.

```
/Users/sindresorhus/dev/ava/node_modules/babel-core/lib/transformation/file/options/option-manager.js:193
          throw new ReferenceError(messages.get("pluginUnknown", plugin, loc, i, dirname));
          ^

ReferenceError: Unknown plugin "transform-runtime" specified in "base" at 1, attempted to resolve relative to "/Users/sindresorhus/dev/boxen"
    at /Users/sindresorhus/dev/ava/node_modules/babel-core/lib/transformation/file/options/option-manager.js:193:17
    at Array.map (native)
    at Function.normalisePlugins (/Users/sindresorhus/dev/ava/node_modules/babel-core/lib/transformation/file/options/option-manager.js:173:20)
    at OptionManager.mergeOptions (/Users/sindresorhus/dev/ava/node_modules/babel-core/lib/transformation/file/options/option-manager.js:271:36)
    at OptionManager.init (/Users/sindresorhus/dev/ava/node_modules/babel-core/lib/transformation/file/options/option-manager.js:416:10)
    at File.initOptions (/Users/sindresorhus/dev/ava/node_modules/babel-core/lib/transformation/file/index.js:191:75)
    at new File (/Users/sindresorhus/dev/ava/node_modules/babel-core/lib/transformation/file/index.js:122:22)
    at Pipeline.transform (/Users/sindresorhus/dev/ava/node_modules/babel-core/lib/transformation/pipeline.js:42:16)
    at Object.transformFileSync (/Users/sindresorhus/dev/ava/node_modules/babel-core/lib/api/node.js:116:10)
    at Object.<anonymous> (/Users/sindresorhus/dev/ava/lib/babel.js:73:24)
  ✖ /Users/sindresorhus/dev/boxen/test.js exited with a non-zero exit code: 1
npm ERR! Test failed.  See above for more details.
```

// @jamestalmage

--- a/lib/babel.js
+++ b/lib/babel.js
@@ -52,8 +52,8 @@ var powerAssert = createEspowerPlugin(babel, {
 
 // if generators are not supported, use regenerator
 var options = {
-	presets: ['stage-2', 'es2015'],
-	plugins: [powerAssert, 'transform-runtime'],
+	presets: [require('babel-preset-stage-2'), require('babel-preset-es2015')],
+	plugins: [powerAssert, require('babel-plugin-transform-runtime')],
 	sourceMaps: true
 };
 

